app/src/ui/thank-you/thank-you.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react'
+
+import { encodePathAsUrl } from '../../lib/path'
+
+import { ReleaseNote } from '../../models/release-notes'
+
+import { Dialog, DialogContent } from '../dialog'
+
+import { RichText } from '../lib/rich-text'
+import { Repository } from '../../models/repository'
+import { getDotComAPIEndpoint } from '../../lib/api'
+import { GitHubRepository } from '../../models/github-repository'
+import { Owner } from '../../models/owner'
+
+// HACK: This is needed because the `Rich`Text` component
+// needs to know what repo to link issues against.
+// Since release notes are Desktop specific, we can't
+// rely on the repo info we keep in state, so we've
+// stubbed out this repo
+const desktopOwner = new Owner('desktop', getDotComAPIEndpoint(), -1)
+const desktopUrl = 'https://github.com/desktop/desktop'
+const desktopRepository = new Repository(
+  '',
+  -1,
+  new GitHubRepository('desktop', desktopOwner, -1, false, desktopUrl),
+  true
+)
+
+const ReleaseNoteHeaderLeftUri = encodePathAsUrl(
+  __dirname,
+  'static/release-note-header-left.svg'
+)
+const ReleaseNoteHeaderRightUri = encodePathAsUrl(
+  __dirname,
+  'static/release-note-header-right.svg'
+)
```

Comment:
Would it be weird if we extracted this (duplicated in `release-notes-dialog.tsx`) to a common place?

Reply:
Yeah I agree that'd be nice if possible! 👍

Reply:
Yep. Good point!